alias.go

```diff
@@ -10,10 +10,15 @@ type FunctionAliasMap = map[string][]string
 // It should be called after all functions and aliases have been added and
 // inside the Build function in case of using a custom handler.
 func AssignAliases(h Handler) {
-	for originalFunction, aliases := range h.Aliases() {
+	for originalName, aliases := range h.RawAliases() {
+		_, exists := h.RawFunctions()[originalName]
+		if !exists {
+			continue
+		}
+
 		for _, alias := range aliases {
-			if fn, ok := h.Functions()[originalFunction]; ok {
-				h.Functions()[alias] = fn
+			if fn, ok := h.RawFunctions()[originalName]; ok {
+				h.RawFunctions()[alias] = fn
 			}
 		}
 	}
```

Comment:
There is something strange to me you are checking if `h.RawFunctions()[originalName]` exists, but then you don't get the value

But then in the loop you look for `h.RawFunctions()[originalName]` to check if it exists (again) and then get its value